src/components/Title/index.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styles from "./index.module.css";
+
+const THEMES_VALUES = [
+  styles.redTheme,
+  styles.neonCarrotTheme,
+  styles.greyishBlueTheme,
+];
+
+const TYPE_VALUES = [styles.h1, styles.h2];
+
+export const THEMES = {
+  RED: THEMES_VALUES[0],
+  NEON_CARROT: THEMES_VALUES[1],
+  GREISH_BLUE: THEMES_VALUES[2],
+};
```

Comment:
I thought someone would show me a better solution 😄

Reply:
I was thinking in use classnames packages. what do you think?

Reply:
Sounds good, but how will that help exporting the configurations?

Reply:
What do you mean by exporting the configurations?

Reply:
I think it would be nice to export the actual options for the `themes` prop, but it might be a thing for me only. Your call